operations/repo.go

```diff
@@ -46,7 +84,55 @@ func Repo() cli.Command {
 	}
 }
 
-func repoFlags() []cli.Flag {
+func repoSubmit() cli.Command {
+	return cli.Command{
+		Name:  "submit",
+		Usage: "submit a repobuilder job to a remote service",
+		Flags: repoFlags(
+			cli.StringSliceFlag{
+				Name:  "packages",
+				Usage: "path to packages, searches for valid packages recursively",
+			},
+			cli.StringFlag{
+				Name:  "service",
+				Usage: "specify the path to a repobuilder service",
+				Value: "https://barque.mongodb.comm",
+			},
+			cli.StringFlag{
+				Name:   "username",
+				Usage:  "specify the username for a user to authenticate to the repobuilding service",
+				EnvVar: "BARQUE_USERNAME",
+			},
+			cli.StringFlag{
+				Name:   "password",
+				Usage:  "specify the password to authenticate to the repobuilding service",
+				EnvVar: "BARQUE_PASSWORD",
+			},
+		),
+		Action: func(c *cli.Context) error {
+			ctx, cancel := ctxWithTimeout(c.Duration("timeout"))
+			defer cancel()
+
+			grip.Infof("curator version: %s", curator.BuildRevision)
```

Comment:
Do we need this print?

Reply:
nah, but the local-mode version does it and I think having a clearer idea of what version of curator is running (to ease debugging when -latest is used) seems reasonable.